Tangem/Modules/Main/Utilities/MainNotificationsObserver.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  MainNotificationsObserver.swift
+//  Tangem
+//
+//  Created by Andrey Fedorov on 07.12.2023.
+//  Copyright © 2023 Tangem AG. All rights reserved.
+//
+
+import Foundation
+
+protocol MainNotificationsObserver: AnyObject {
+    func didChangeNotificationInputs(_ inputs: [NotificationViewInput], forUserWalletWithId userWalletId: UserWalletId)
```

Comment:
```suggestion
    func didChangeNotificationInputs(_ inputs: [NotificationViewInput], for userWalletId: UserWalletId)
```

тогда при вызове будет:
``` swift
observer.didChangeNotificationInputs(inputs, for: userWalletModel.userWalletId)
```
Мне кажется так покрасивше

Reply:
Заменил, видимо раньше я юзал тут builtin тип (типа Data или Int), для которых рекомендуется задавать больше инфы в arg label ¯\_(ツ)_/¯